webpack.config.js

```diff
@@ -105,6 +105,14 @@ const config = {
                     'react-focus-lock': `${__dirname}/node_modules/react-focus-lock`
                 }
             }
+        }, {
+            test: /\.svg$/,
+            use: [ {
+                loader: '@svgr/webpack',
```

Comment:
The react-native SVG package claims to also work on web. Any chance we can use that transformer and use it here instead of depending on a different loader for web?